index.html

```diff
@@ -43,6 +43,142 @@ <h1>
         <li><a href="#"><img src="images/Vector.svg" alt="Vector"></a></li>
       </ul>
     </section>
+    <div class="works">
+      <article class="card-works">
+        <img class="card-image" src="images/works/project_01.main_snapshot.jpg" alt="Main snapshot of project 1">
+        <div class="card-details">
```

Comment:
- [ ] Please use a `section` gat for this `.works` section instead of using a generic `div` tag.

Reply:
Thank you for your feedback, Brian!
The reason I changed from <section> to a regular <div> was a linting warning complaining about 'no heading for this section'. Can you please suggest how should I treat this issue? A hidden heading would be fine, or simply ignoring the linting warning is preferred?